### ISSUE TITLE
Tipo de dato ID

### ISSUE THREAD

**Issue by facundokrens65049** [PI-ITBA/2024_01#398]
Hola!
Se puede considerar que los ID de las infracciones son siempre números? Para ambas ciudades son números, pero quizá para otras no...

**Comment by marcelogarberoglio** [comment]
En 4 partes del enunciado figura  "número entero no negativo"
![image](https://github.com/PI-ITBA/2024_01/assets/13138167/614016dd-d7d0-4aa3-a178-a85f854d5d01)

**Comment by facundokrens65049** [comment]
Sí, pero dentro de la descripción de los datos de cada ciudad. Eso sería algo particular de esas ciudades, o podemos asumir que siempre se cumple?
